### ISSUE TITLE
Focus not returning to dropdown on attribute change - single product page

### ISSUE THREAD

**Issue by ArthurThoby** [woocommerce/woocommerce#24128]
**Describe the bug**
Focus is not returned to the drop-down attribute list after a new variation is chosen (onChange). Rather, the focus goes back to the flexslider gallery navigation ('flex-control-nav').

This is against accessibility guidelines and ruins the experience for tab navigation; see [w3 guidelines](https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html).

**To Reproduce**
1. Go to any single product page for a variable product
2. Click on 'size' drop-down or any other other attribute
3. Change attribute to a new value
4. See error (use left or right keys to see where the focus jumped)

**Expected behavior**
Focus should be returned to the drop-down after new variation is selected.

**Isolating the problem:**
- [x] This bug happens with [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.
- [x] I tracked the problem with the chrome coadd-to-cart-variations.min.jsnsole debugger and found out it is related to the onChange function in 

**Possible fix**
Simply add `this.focus()` to the end of the onChange function in add-to-cart-variations.min.js

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://localhost/wordpress
Site address (URL): https://localhost/wordpress
WC Version: 3.6.5
Log Directory Writable: ✔
WP Version: 5.2.2
WP Multisite: –
WP Memory Limit: ❌ 40 MB - We recommend setting memory to at least 64MB. See: Increasing memory allocated to PHP
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.2.16
PHP Post Max Size: 500 MB
PHP Time Limit: 120
PHP Max Input Vars: 1000
cURL Version: 7.64.0
OpenSSL/1.1.1b

SUHOSIN Installed: –
MySQL Version: 8.0.15
Max Upload Size: 400 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ Your server does not have the SoapClient class enabled - some gateway plugins which use SOAP may not work as expected.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.6.5
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Total Database Size: 7.14MB
Database Data Size: 4.88MB
Database Index Size: 2.26MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_comments: Data: 0.06MB + Index: 0.09MB
wp_icl_content_status: Data: 0.02MB + Index: 0.02MB
wp_icl_core_status: Data: 0.02MB + Index: 0.02MB
wp_icl_flags: Data: 0.02MB + Index: 0.02MB
wp_icl_languages: Data: 0.02MB + Index: 0.03MB
wp_icl_languages_translations: Data: 0.19MB + Index: 0.09MB
wp_icl_locale_map: Data: 0.02MB + Index: 0.00MB
wp_icl_message_status: Data: 0.02MB + Index: 0.03MB
wp_icl_mo_files_domains: Data: 0.02MB + Index: 0.02MB
wp_icl_node: Data: 0.02MB + Index: 0.00MB
wp_icl_reminders: Data: 0.02MB + Index: 0.00MB
wp_icl_string_packages: Data: 0.02MB + Index: 0.00MB
wp_icl_string_pages: Data: 0.09MB + Index: 0.06MB
wp_icl_string_positions: Data: 0.02MB + Index: 0.02MB
wp_icl_string_status: Data: 0.02MB + Index: 0.02MB
wp_icl_string_translations: Data: 0.14MB + Index: 0.05MB
wp_icl_string_urls: Data: 0.02MB + Index: 0.02MB
wp_icl_strings: Data: 0.52MB + Index: 0.48MB
wp_icl_translate: Data: 0.02MB + Index: 0.02MB
wp_icl_translate_job: Data: 0.02MB + Index: 0.02MB
wp_icl_translation_batches: Data: 0.02MB + Index: 0.00MB
wp_icl_translation_status: Data: 0.03MB + Index: 0.02MB
wp_icl_translations: Data: 0.02MB + Index: 0.06MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 2.50MB + Index: 0.05MB
wp_postmeta: Data: 0.28MB + Index: 0.09MB
wp_posts: Data: 0.09MB + Index: 0.06MB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.02MB + Index: 0.03MB
wp_usermeta: Data: 0.02MB + Index: 0.03MB
wp_users: Data: 0.02MB + Index: 0.05MB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB

### Post Type Counts ###

attachment: 53
nav_menu_item: 25
page: 11
post: 1
product: 3
product_variation: 7
revision: 18
scheduled-action: 118
shop_coupon: 1
shop_order: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (20) ###

All In One SEO Pack: by Michael Torbert – 3.1.1
All-in-One WP Migration: by ServMask – 6.97
Image Regenerate & Select Crop: by Iulia Cazan – 4.7.4
Jetpack by WordPress.com: by Automattic – 7.5.2
lazysizes: by Patrick Sletvold – 1.1.0
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.1
WPML Multilingual CMS: by OnTheGoSystems – 4.2.4.1
WooCommerce Order Test: by WP Fix It – 1.5 – Not tested with the active version of WooCommerce
WooCommerce Admin: by WooCommerce – 0.14.0
WooCommerce Stripe Gateway: by WooCommerce – 4.2.2
WooCommerce Multilingual: by OnTheGoSystems – 4.6.5
WooCommerce Services: by Automattic – 1.21.0
WooCommerce: by Automattic – 3.6.5
WordPress Importer: by wordpressdotorg – 0.6.4
WP-DBManager: by Lester 'GaMerZ' Chan – 2.79.2
WP PHP Console: by Fulvio Notarstefano – 1.5.4
WP Retina 2x: by Jordy Meow – 5.5.6
WPML Media: by OnTheGoSystems – 2.5.1
WPML String Translation: by OnTheGoSystems – 2.10.2
WPML Translation Management: by OnTheGoSystems – 2.8.3

### Inactive Plugins (4) ###

Google Analytics for WordPress by MonsterInsights: by MonsterInsights – 7.7.1
Simple Tags: by Amaury BALMER – 2.5.4
WC Variations Radio Buttons: by 8manos – 2.0.2 – Not tested with the active version of WooCommerce
WP Mail SMTP: by WPForms – 1.5.0

### Settings ###

API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: right
Thousand Separator: 
Decimal Separator: ,
Number of Decimals: 0
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #18 - /shop/
Cart: #23 - /basket/
Checkout: #24 - /checkout/
My account: #25 - /my-account/
Terms and conditions: ❌ Page visibility should be public

### Theme ###

Name: Storefront Child Theme
Version: 1.0.0
Author URL: http://stuartduff.com
Child Theme: ✔
Parent Theme Name: Storefront
Parent Theme Version: 2.5.1
Parent Theme Author URL: https://woocommerce.com/
WooCommerce Support: ✔

### Templates ###

Overrides: storefront-child-theme-master-test/woocommerce/cart\mini-cart.php
storefront-child-theme-master-test/woocommerce/content-single-product.php
storefront-child-theme-master-test/woocommerce/global\quantity-input.php
storefront-child-theme-master-test/woocommerce/single-product\price.php
storefront-child-theme-master-test/woocommerce/single-product\product-image.php
storefront-child-theme-master-test/woocommerce/single-product\product-thumbnails.php
storefront-child-theme-master-test/woocommerce/single-product\sale-flash.php
storefront-child-theme-master-test/woocommerce/single-product\tabs\description.php
storefront-child-theme-master-test/woocommerce/single-product\tabs\tabs.php


### Action Scheduler ###

Complete: 115
Oldest: 2019-06-10 18:53:22 +0200
Newest: 2019-07-11 17:33:41 +0200

Pending: 3
Oldest: 2019-07-11 18:33:41 +0200
Newest: 2019-07-12 17:18:06 +0200

Canceled: 0
Oldest: –
Newest: –

In-progress: 0
Oldest: –
Newest: –

Failed: 0
Oldest: –
Newest: –

```
</details>

**Comment by vedanshujain** [comment]
Lets continue the discussion at #24339
